scripts/src/main/resources/scripts/command/ide

```diff
@@ -173,9 +173,34 @@ function doUpdateCliScript() {
   fi
 }
 
+function doInstallAutocompletion() {
+  # Install autocompletion
+  devon_autocomplete="${DEVON_HOME_DIR}/.devon/autocomplete"
+  if [ ! -f "${devon_autocomplete}" ]
+  then
+    cp "${DEVON_IDE_HOME}/scripts/autocomplete" "${devon_autocomplete}"
+    AUTOCOMPLETION="source ~/.devon/autocomplete"
+    echo -e "${AUTOCOMPLETION}" >> ~/.bashrc
+    if ! grep -q "compinit" ~/.zshrc
+    then
+      echo -e 'autoload -Uz compinit\ncompinit' >> ~/.zshrc
+    fi
+    if ! grep -q "bashcompinit" ~/.zshrc
+    then
+      echo -e 'autoload bashcompinit\nbashcompinit' >> ~/.zshrc
+    fi
+    echo -e "${AUTOCOMPLETION}" >> ~/.zshrc
+    if doIsMacOs
+    then
+      echo -e "${AUTOCOMPLETION}" >> ~/.bash_profile
+    fi
```

Comment:
why do we need this on MacOS? It do not see we need this at all or why it is OS specific...

Reply:
This is OS specific. Mac uses `.bash_profile` and Linux use `.bashrc`. I added for this autocompletion `.bash_profile`to the code that it also runs with MacOS. We can also add a condition for Linux, because it is also not useful for mac, but maybe for windows. This is the reason why only add the condition for  `.bash_profile`, because it is only for MacOS.